### ISSUE TITLE
".label" in operands is interpreted as 0

### ISSUE THREAD

**Issue by ZornsLemma** [stardot/beebasm#44]
```
$ cat test.6502
	ORG &2000
.start

.loop
	JMP loop
	JMP .loop

.end

	SAVE "TEST", start, end
$ beebasm -i test.6502 -v
.start
.loop
     2000   4C 00 20   JMP &2000
     2003   4C 00 00   JMP &0000 <------------------------ bug??
.end
Saving file 'TEST'
Processed file 'test.6502' ok
$ xxd TEST
00000000: 4c00 204c 0000                           L. L..
```
mungre has a fix for this: https://github.com/stardot/beebasm/tree/mungre/badfpfix

(See discussion on stardot at https://stardot.org.uk/forums/viewtopic.php?f=55&t=12199&sid=0e459a92dd912c3335c0cd3425b50e80&start=90#p287960)

**Comment by cardboardguru76** [comment]
This bug it me twice on the first day of using BeebAsm. 
1) Because as a newbie, I first assumed the dot was part of the label, rather then just a way of defining one.
2) Then later, I copied an assembler snippet intended for another assembler, and it DID use a dot as part of the label.
Between the two, this cost me a couple of hours of time wondering why my seemingly correct code didn't work.

Just pointing out that people do hit this bug often, to encourage this fix to be applied and released.

**Comment by ZornsLemma** [comment]
I've just merged this fix to proposed-updates.

mungre - apologies, I accidentally pushed a README tweak mentioning this onto your branch...

**Comment by mungre** [comment]
Fixed in v1.10 and [has a test](https://github.com/stardot/beebasm/blob/master/test/1-values/baddec1.fail.6502).
